core/blockchain.go

```diff
@@ -176,35 +204,76 @@ func ExecuteBlockEphemerallyForBSC(
 		return nil, fmt.Errorf("writing changesets for block %d failed: %w", header.Number.Uint64(), err)
 	}
 
-	return receipts, nil
+	var logs []*types.Log
+	for _, receipt := range receipts {
+		logs = append(logs, receipt.Logs...)
+	}
+
+	blockLogs := ibs.Logs()
+	var stateSyncReceipt *types.ReceiptForStorage
+	if len(blockLogs) > 0 {
```

Comment:
Looks like copy-paste from the other function, `ExecuteBlockEphemerally`. Is this actually required for BSC? I think this is only beed for Bor consensus

Reply:
Ok, I will check it out once it is removed